### ISSUE TITLE
it is out of order app maker

### ISSUE THREAD

**Issue by farkasember27** [IgorTimofeev/MineOS#533]
it is out of order app maker (project ozone 3)
![2023-03-23_15 49 46](https://user-images.githubusercontent.com/54159181/227246384-3c62cbd5-aafb-4fc8-ac24-83cd090340c2.png)

**Comment by PublicVoidUpdate** [comment]
Already covered in other issues, app market is down for the near future. The owner has a copy on his computer but he can't start up a new server until he gets to the computer. And since he's on vacation that won't be until he's off. It won't be a permanent thing, but for a little bit there's not anything we can do.

**Comment by IgorTimofeev** [comment]
It's ok #531
